requirements.txt

```diff
@@ -1,32 +1,33 @@
-autopep8==1.3.4
-boto3==1.5.22
-coverage==4.5
-cryptography==2.1.4
-decorator==4.2.1
-Django==1.11.10
+autopep8==1.3.5
+boto3==1.9.10
+coverage==5.0a2
+cryptography==2.3.1
+decorator==4.3.0
+Django==1.11.15
 django-annoying==0.10.4
-django-markdown2==0.3.0
-django-rest-swagger==2.1.2
-djangorestframework==3.7.7
-et3==1.5
-google-api-python-client==1.6.5
-green==2.12.0
+django-markdown2==0.3.1
+django-rest-swagger==2.2.0
+djangorestframework==3.8.2
+et3==1.5.1
+google-api-python-client==1.7.4
+green==2.12.1
 kids.cache==0.0.7
 markdown2==2.3.5
-newrelic==2.100.0.84
-oauth2client==4.1.2
-psycopg2==2.7.3.2
-pyflakes==1.6.0
-pylint==1.8.2
-pylint-django==0.9.0
-pyOpenSSL==17.5.0
-python-dateutil==2.6.1
-python-json-logger==0.1.8
-pytz==2017.3
-PyYAML==3.12
-requests==2.18.4
+newrelic==4.4.1.104
```

Comment:
should keep an eye on that

Reply:
yes, I noticed that huge jump and wondered just how far behind everything else was. bot-lax got a bump recently, lax and observer could do with some TLC too.

Django has adopted a LTS release cycle so the latest 1.xx line is supported until 2021, so no need for anything major in lax/metrics/observer/dash2.0 for a bit